Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM ccnmtl/django.base
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# install node stuff
-COPY package.json /node/
-RUN cd /node && npm install && touch /node/node_modules/sentinal
+# syntax=docker/dockerfile:1
+FROM python:3.9
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+EXPOSE 8000
+WORKDIR /src
+COPY requirements.txt /src/
+RUN apt-get update
+# RUN apt-get install -y libgdal-dev
+RUN apt-get install -y \ 
+    gdal-bin \
+    libgdal-dev
 
-# build virtualenv and run tests
-ADD wheelhouse /wheelhouse
-RUN /ve/bin/pip install --no-index -f /wheelhouse -r /wheelhouse/requirements.txt \
-    && rm -rf /wheelhouse && touch /ve/sentinal
-WORKDIR /app
-COPY . /app/
-RUN VE=/ve/ MANAGE="/ve/bin/python manage.py" NODE_MODULES=/node/node_modules/ make
+# Set environment variables required by GDAL
+ENV CPLUS_INCLUDE_PATH=/usr/include/gdal
+ENV C_INCLUDE_PATH=/usr/include/gdal
 
-EXPOSE 8000
-ADD docker-run.sh /run.sh
-ENV APP locustempus
-ENTRYPOINT ["/run.sh"]
-CMD ["run"]
+RUN pip install --upgrade pip
+RUN pip install wheel setuptools
+RUN pip install psycopg2-binary
+RUN pip install GDAL==3.2.2.1
```

Comment:
If the `GDAL` library is required for locustempus, then it should be required in the main requirements.txt. The docker container's pip environment shouldn't have to diverge too much from the non-docker pip environment, as there's no real difference in these environments from the application's perspective.